이호은/boj15828.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from collections import deque
+
+dq = deque()
+
+SIZE = int(sys.stdin.readline())
+
+while True:
+    N = int(sys.stdin.readline())
+    if N == -1:
+        break
+    elif N == 0:
+        dq.popleft()
+    else:
+        if len(dq) < SIZE:
+            dq.append(N)
```

Comment:
이게 정답은 아니지만 바로 끊어지는 케이스는 일찍 return 하는게 요즘 좋은 패턴인듯요
```suggestion
    if N == -1:
        break
    if N == 0:
        dq.popleft()
    else:
        if len(dq) < SIZE:
            dq.append(N)
```